range_sensor_layer/include/range_sensor_layer/range_sensor_layer.h

```diff
@@ -65,6 +67,7 @@ class RangeSensorLayer : public costmap_2d::CostmapLayer
   boost::function<void(sensor_msgs::Range& range_message)> processRangeMessageFunc_;
   boost::mutex range_message_mutex_;
   std::list<sensor_msgs::Range> range_msgs_buffer_;
+  std::map<std::pair<int, int>, double> point_map;
```

Comment:
A) Please use the trailing underscore format for private variables
B) Did you choose signed ints over unsigned ints for a reason?
C) From the variable name, it is unclear what the values represent.